005-RockPaperScissorsBoom/Student/Challenge-07.md

```diff
@@ -0,0 +1,28 @@
+# Challenge 07 - Leverage SignalR
+
+[< Previous Challenge](./Challenge-06.md) - **[Home](../README.md)** - [Next Challenge >](./Challenge-08.md)
+
+## Introduction
+
+With this challenge you will deploy and add your own `SmartBot` by leveraging `SignalR`, you will add a new competitor in your games.
+
+## Description
+
+- There is already the `RockPaperScissorsBoom.ExampleBot` project in your solution implementing a `SignalR` bot, let's just use it and deploy it!
+- Deploy it on a new Azure Web App for Containers instance.
+
+## Success Criteria
+
+To complete this challenge successfully, you should be able to:
+
+- Validate that your new Azure Web App & Docker image are deployed using `az webapp list` and `az cr repository show-tags`.
+- In your web browser, navigate to the main web app (Server), add this Bot as a new competitor and play a game, make sure it's working without any error.
+
+## Learning Resources
+
+- [SignalR](https://dotnet.microsoft.com/en-us/apps/aspnet/signalr)
+
+## Tips
+
+- Revisit challenges [Challenge 4 - Run the app on Azure](RunOnAzure.md) and/or [Challenge 7 - Build a CI/CD pipeline with Azure DevOps](BuildCICDPipelineWithAzureDevOps.md). For the latter, the recommendation is to create its own Build and Release definition (not impacting the ones already created for the `Server`).
```

Comment:
Were the challenges shuffled around/deleted?  The challenge names and links don't match up anymore (Challenge 4 and 7).

Reply:
yes, I had to rename & move them around, I missed this reference, fixing...